_docs/expressions.md

```diff
@@ -104,6 +104,16 @@ This will generate a rule that fires if the Customer fact has a vip status, **or
 
 For further information, see [Boolean Expressions](/docs/booleans).
 
+## Exists Expressions
+
+Exists expresssions only test for the existence or absence of matching facts without triggering for every instance of the fact.  For example:
+
+{% highlight clojure %}
+[:exists [Person (= name "Bob")]]
+{% endhighlight %}
+
+This will trigger the _first_ time a Person with the name "Bob" is inserted, but not the second or third.  It will only retract after all supporting facts are removed.
+
```

Comment:
I'd suggest noting that the :exists node can fire more than once as defined by the underlying accumulator rules around firing once per distinct set of bindings.  I don't think you need to enumerate every case here specifically, I'd just refer back to the accumulator documentation.

Example:
````
clara.test-rules> (defrule the-rule
                    [:exists [Temperature (= ?loc location)]]
                    =>
                    (insert! (->WindSpeed 0 ?loc)))
#'clara.test-rules/the-rule
clara.test-rules> (defquery the-query [] [?w <- WindSpeed])
#'clara.test-rules/the-query
clara.test-rules> (-> (mk-session [the-rule the-query] :cache false) (insert (->Temperature 0 "LGW") (->Temperature 0 "LHR")) fire-rules (query the-query))
({:?w #clara.rules.testfacts.WindSpeed{:windspeed 0, :location "LGW"}} {:?w #clara.rules.testfacts.WindSpeed{:windspeed 0, :location "LHR"}})
clara.test-rules> (-> (mk-session [the-rule the-query] :cache false) (insert (->Temperature 0 "LGW") (->Temperature 0 "LGW")) fire-rules (query the-query))
({:?w #clara.rules.testfacts.WindSpeed{:windspeed 0, :location "LGW"}})
````

Otherwise this looks good.

Reply:
Done.